index.html

```diff
@@ -0,0 +1,219 @@
+                        <li class="css"><img class="platform_img" src="./img/css.png" alt="css picture">CSS</li>
+                    </ul>
+                    <div class="total__wrapper">
+                        <div class="platform__wrapper">
+                            <h3 class="frameworks-title">Frameworks</h3>
+                            <ul class="bar__wrapper">
+                                <li class="bar"></li>
+                                <li class="bar"></li>
+                            </ul>
+                        </div>
+                        <div class="underline"></div>
+                        <div class="platform__wrapper">
+                            <h3 class="frameworks-title">Skills</h3>
+                            <ul class="bar__wrapper">
+                                <li class="bar"></li>
+                                <li class="bar"></li>
+                            </ul>
+                        </div>
+                        <div class="underline"></div>
+                    </div>         
+                </div>
+            </div>
+            
+        </div>
+        
+    </section>
+
+    <section class="contact-me_page">
+        <div class="contact__wrapper">
+            <h2 class="contact__header">Contact me</h2>
+            <p class="description__contact">I am a software developer! I can help you build a product, feature or website look through some of my work and experiance! if you like what you see or have a project you need coded don't hesitate to contact me. </p>
+            <form>
+                <input class="text__input" type="text" placeholder="Addisalem Seifu"><br>
+                <input class="text__input" type="email" placeholder="addisalemseifu@gmail.com"><br>
+                <textarea name="text" id="message__box" cols="30" rows="5" placeholder="write your message hear"></textarea><br>
+                <input class="submit__btn" type="submit">
+            </form>
+        </div>
+    </section>
+
+    <section class="forms__page">
+        <div class="form__wrapper">
+            <h3 class="form__title">Contact me</h3>
+            <p class="form__paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <form class="main__form" action="https://formspree.io/f/xknkypba" method="post">
+                <input name="name" id="name" class="input__space" type="text" placeholder="Enter your Name" maxlength="30" required>
+                <input name="email" id="email" class="input__space" type="email" placeholder="Email adress" required>
+                <textarea class="text__area" name="message" id="message" cols="30" rows="10" placeholder="write your message here" maxlength="500" required></textarea>
+                <button class="input__btn" type="submit">Get it Touch</button>
+            </form>
+        </div>
+    </section>
```

Comment:
Please remove extra space from the bottom of the contact form. Kindly see the screenshot below for reference.
![extra](https://user-images.githubusercontent.com/35504149/134526997-8d2e839a-e67b-432e-80ca-912d2f8751fe.png)